java/java.source.nbjavac/src/org/netbeans/modules/java/source/nbjavac/parsing/TreeLoader.java

```diff
@@ -101,7 +101,9 @@
     private static final ThreadLocal<Boolean> isTreeLoading = new ThreadLocal<Boolean>();
 
     public static void preRegister(final Context context, final ClasspathInfo cpInfo, final boolean detached) {
-        context.put(lazyTreeLoaderKey, new TreeLoader(context, cpInfo, detached));
+        LazyTreeLoader instance = context.get(lazyTreeLoaderKey);
```

Comment:
This check is required otherwise we will get below error

	java.lang.AssertionError: duplicate context value
	at com.sun.tools.javac.util.Context.put(Context.java:141)
	at org.netbeans.modules.java.source.nbjavac.parsing.TreeLoader.preRegister(TreeLoader.java:106)
	at org.netbeans.modules.java.source.nbjavac.parsing.TreeLoader$TreeLoaderRegistryImpl.enhance(TreeLoader.java:579)
	at org.netbeans.modules.java.source.parsing.JavacParser.lambda$createJavacTask$10(JavacParser.java:796)